addons/website_sale_collect/controllers/main.py

```diff
@@ -29,6 +29,16 @@ def _prepare_checkout_page_values(self, order_sudo, **query_params):
         """ Override of `website_sale` to include the unavailable products for the selected pickup
         location. """
         res = super()._prepare_checkout_page_values(order_sudo, **query_params)
+        available_dms = order_sudo._get_delivery_methods()
+
+        for delivery_method in available_dms:
```

Comment:
why are you checking all available delivery method. Shouldn't we only need to check only pay_in_store delivery methods :thinking: